tapir/statistics/models.py

```diff
@@ -25,7 +32,13 @@ class PurchaseBasket(models.Model):
     discount = models.FloatField()  # Rabatt_SUM
 
 
-class FancyGraphCache(models.Model):
-    view_name = models.CharField(max_length=255)
-    date = models.DateField()
-    value = models.IntegerField()
```

Comment:
Did you remove this on purpose?

Reply:
Oh no, sorry. Justed wanted to get rid of it from my branch. Deleted by mistake.